core/src/main/scala/stryker4s/run/ProcessMutantRunner.scala

```diff
@@ -52,9 +52,9 @@ class ProcessMutantRunner(command: Command, process: ProcessRunner)(implicit con
       mutatedFile <- mutatedFiles
       subPath = mutatedFile.fileOrigin.relativePath
       mutant <- mutatedFile.mutants
+      id = mutatedFiles.flatMap(_.mutants).toSeq.indexOf(mutant) + 1
```

Comment:
This search is O(n^2), right? For every mutant the entire list is searched. Something like `zipWithIndex` might be better

Reply:
Agreed zipWithIndex would be a lot better. We could simply do something like: `(mutatedFile, index) <- mutatedFiles.zipWithIndex`. After that change the log statement to use the index and not touch the current Id i think hugo is right and that it will not run the proper mutations otherwise.